Description: disable SSLv3 after removal from OpenSSL
 For background see #797926. Thanks to Kurt Roeckx for pointer.
Author: Daniel Stender <debian@danielstender.com>
Bug: https://bugs.debian.org/803946
Forwarded: not-needed
Last-Update: 2015-11-17

--- a/SWIG/_ssl.i
+++ b/SWIG/_ssl.i
@@ -56,8 +56,10 @@
 %rename(sslv2_method) SSLv2_method;
 extern SSL_METHOD *SSLv2_method(void);
 #endif
+#ifndef OPENSSL_NO_SSL3
 %rename(sslv3_method) SSLv3_method;
 extern SSL_METHOD *SSLv3_method(void);
+#endif
 %rename(sslv23_method) SSLv23_method;
 extern SSL_METHOD *SSLv23_method(void);
 %rename(tlsv1_method) TLSv1_method;
